README.md

```diff
@@ -16,11 +16,12 @@ educators to increase student engagement and make learning fun.
 ### System Requirements
 
 1. Ruby (>= 2.3.1)
-2. Ruby on Rails
-3. PostgreSQL
-4. ImageMagick or GraphicsMagick (For [MiniMagick](https://github.com/minimagick/minimagick))
-5. Node.js
-6. Yarn
+1. Ruby on Rails
+1. PostgreSQL
+1. ImageMagick or GraphicsMagick (For [MiniMagick](https://github.com/minimagick/minimagick))
+1. Ghostscript (for PDF to image processing for [MiniMagick](https://github.com/minimagick/minimagick))
```

Comment:
Actually, the scribing tests passed on my local also.  Maybe merge this with the line above ? cause it's imagemagick specific.